### DISCUSSION TITLE
OPENAM and LDAP setup issue .

### DISCUSSION THREAD

**Discussion by bharats22** [OpenIdentityPlatform/OpenAM#537] [General]
1) Want to know if generic LDAP is compatible with OpenAM (tomcat) ? 

2) We are receiving "There is no information to display for this entry" message on OPENAM while accessing subjects . We need to know if there is any log where we can see more information on this issue or how can we troubleshoot this error message .

3) Someone suggested to use LDP.exe to get more information on this issue , but we are unable to find a website from which we can download LDP.exe. If not then is there any alternate to LDP.exe for connection testing ?

**Comment by maximthomas** [comment]
@bharats22 do you want to user LDAP for OpenAM configuration store or user store? What do you see in OpenAM logs?

**Comment by bharats22** [comment]
I think it is user/Data store . We are new to OPENAM and dont know where the logs will be present . Your guidance is much appreciated .

**Reply by maximthomas** [reply]
The logs should be here: `$HOME/openam/openam/debug `

**Comment by bharats22** [comment]
We have a jar file that we deployed for OPENAM so we dont have the above mentioned path .

**Reply by maximthomas** [reply]
@bharats22  please provide more details about you setup and configuration and what you are trying to achieve.
